src/sessions/PageManager.ts

```diff
@@ -1,14 +1,22 @@
 import { Config } from '../orchestration/Orchestration';
 import { RecordEvent } from '../plugins/Plugin';
 import { PageViewEvent } from '../events/page-view-event';
+import { NavigationEvent } from '../events/navigation-event';
+import { PERFORMANCE_NAVIGATION_EVENT_TYPE } from '../plugins/utils/constant';
+import { MonkeyPatch, MonkeyPatched } from '../plugins/MonkeyPatched';
 
+type Fetch = typeof fetch;
+type Send = () => void;
 export const PAGE_VIEW_TYPE = 'com.amazon.rum.page_view_event';
 
 export type Page = {
     pageId: string;
     interaction: number;
     parentPageId?: string;
     start: number;
+    ongoingActivity: Set<XMLHttpRequest>;
+    latestEndTime: number;
+    isLoaded?: boolean;
```

Comment:
issue: These probably shouldn't be stored inside `Page` objects. `Page` objects are serialized and stored/restored by cookies inside `SessionManager`.

Reply:
Makes sense, the pageManager/helper class can hold these and the functionality can be maintained. Will reflect in next revision.